Automattic/woocommerce-payments
# Bump NodeJS to version 14

Fixes #3849

#### Changes proposed in this Pull Request

Current NodeJS version is 12 which is outdated and the purpose of this PR is to bump the NodeJS version to the latest possible.

I tried to bump version to 16, but some dependencies we depend on haven't updated for a while which caused issues on NodeJS 16. There is a plan to update those dependencies, but for now, the node JS 14 is working fine on environments I tested (The latest Ubuntu and Apple Silicon)









#### Testing instructions





* Make sure that all tests are passing
* Make sure that all E2E tests are passing
* Make sure that you can execute npm scripts without errors

-------------------


**Post merge**
